s3cortxutils/s3confstore/s3confstore/cortx_s3_confstore.py

```diff
@@ -58,6 +58,24 @@ def get_all_keys(self):
     """Get all the key value pairs from confstore."""
     return Conf.get_keys(self.default_index)
 
+  def delete_key(self, key: str, save: bool = False):
```

Comment:
Write up the UTs for newly added functions

Reply:
Done